spec/features/admin_can_edit_an_item_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin', type: :feature do
+  create_category
+  create_brand
+  create_items
+  create_user
+  create_rental
+  create_admin
+
+  feature 'Items' do
+    it 'can edit items' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.